cartographer/io/probability_grid_points_processor.cc

```diff
@@ -80,12 +87,22 @@ ProbabilityGridPointsProcessor::FromDictionary(
                                   dictionary->GetBool("draw_trajectories"))
                                      ? DrawTrajectories::kYes
                                      : DrawTrajectories::kNo;
+  auto output_type = OutputType::kPng;
+  if (dictionary->HasKey("output_type")) {
+    const auto output_type_string = dictionary->GetString("output_type");
+    CHECK(output_type_string == "pbstream" || output_type_string == "png");
+    output_type = output_type_string == "pbstream" ? OutputType::kPbstream
```

Comment:
is it possible to use protobuf's `const EnumValueDescriptor* FindValueByName(const string& name) const` from [descriptor.h](https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.descriptor#EnumDescriptor) ?

Reply:
Maybe I have an misunderstanding here, but the dictionary is created by parsing a lua config and not a protobuf. For lua files I haven't found a implementation for enums.